### PR TITLE
3D Tiles Sandcastle Demos

### PR DESCRIPTION
**Pull request by lilleyse**
Fixes https://github.com/AnalyticalGraphicsInc/cesium/issues/5506

Demos (the first 5 are also in the main Showcases tab):

* **3D Tiles BIM** (Power Plant model from Bentley)
* **3D Tiles Point Cloud** (Chappes Church)
* **3D Tiles Photogrammetry** (AGI building)
  * Some artifacts when `skipLevelOfDetail` is on due to thin roof walls
* **3D Tiles Feature Picking** (NYC CityGML)
* **3D Tiles Feature Styling** (NYC CityGML)
* **3D Tiles Adjust Height** (commonly requested code sample on the forum)
* **3D Tiles Batch Table Hierarchy**
* **3D Tiles Formats** (a simplified version of the original 3D Tiles demo)
* **3D Tiles Inspector**
* **3D Tiles Interactivity** (annotations, print metadata, zoom to, hide feature abilities from the original 3D Tiles demo)
* **3D Tiles Point Cloud Styling**

To do:
* [x] Demos still need attribution. Is a sentence at the top of the demo code sufficient?
* [x] Tilesets are currently hosted on my personal Composer account.

### CODE REVIEW

**Comment by pjcozzi** [comment]
> Demos still need attribution. Is a sentence at the top of the demo code sufficient?

Yes, later these may be embedded in the tileset, https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/55

> Tilesets are currently hosted on my personal Composer account.

Please address this soon so it doesn't delay the 1.35 release tomorrow.  If we can't address this by about noon, we'll ship with these urls and update in the next release.

File: Apps/Sandcastle/gallery/3D Tiles BIM.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer');
+
+var tileset = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
```
Comment (pjcozzi) [inline Apps/Sandcastle/gallery/3D Tiles BIM.html]: Perhaps don't put this one in a body of water?

File: Apps/Sandcastle/gallery/3D Tiles Point Cloud.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer');
+
+var tileset = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
+    url : 'https://beta.cesium.com/api/assets/1448?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiJiODNmMjQzNy1mNGI4LTQ2ZTItOWI0NS0yZDUzMzMyNWMxNzQiLCJpZCI6OSwiaWF0IjoxNDgxODI5ODU0fQ.4fmVtksZ1CAzjPBprJ47uIuK0vr-8CcuDKM7M8a0mF4'
```
Comment (pjcozzi) [inline Apps/Sandcastle/gallery/3D Tiles Point Cloud.html]: @AnimatedRNG this is another good point cloud dataset for your testing.

**Comment by pjcozzi** [comment]
Also please add an indoor example like the San Miguel.  Perhaps @likangning93 can help.

**Comment by pjcozzi** [comment]
Updated CHANGES.md in master for this. 3340d144652744479d6f68138dceb0d35de56162

**Comment by lilleyse** [comment]
Ready with the San Miguel model under the **3D Tiles Interior** name. Thanks for tiling it @likangning93.

**Comment by pjcozzi** [comment]
Looks great!

**Pull request merged**